Update cards-actions.md (#11083)
MicrosoftDocs/msteams-docs: msteams-platform/task-modules-and-cards/cards/cards-actions.md

To include an `imBack` action with an Adaptive Card, include the following details in the `msteams` object:

> [!NOTE]

> > The `value` field is a simple string. There is no support for formatting or hidden characters.

Reviewer: ```suggestion
> The `value` field is a simple string that doesn’t support formatting or hidden characters.
```
